packages/perf/android/build.gradle

```
@@ -63,9 +63,22 @@ project.ext {
   ])
 }
 
+apply from: file("./../../app/android/firebase-json.gradle")
+
+def autoCollectionEnabled = "true"
+
+if (rootProject.ext && rootProject.ext.firebaseJson) {
+  if (rootProject.ext.firebaseJson.isFlagEnabled("perf_auto_collection_enabled") == false) {
```

The code you're cloning had a "fun" problem! #4784 that I fixed by altering the gradle API a bit #4791

This clones the fix for the implementation clone: 
https://github.com/invertase/react-native-firebase/pull/4791/files#diff-4e4bf90ffe1ed3e1706bc42bb35a7816f0db7fe5cff54d5c17f761be59b165b0R71

```suggestion
  if (rootProject.ext.firebaseJson.isFlagEnabled("perf_auto_collection_enabled", true) == false) {
```

I'm surprised this built for you, as I think that gradle API change requires the second argument to isFlagEnabled, it should have kicked out some sort of gradle error?

---

Good catch! I've updated the PR.